src/components/SocialShare/SocialShare.tsx

```diff
@@ -184,6 +185,20 @@ export function SocialShare({
                 </a>
             )}
 
+            {socialNetworks.includes(SocialNetwork.THREADS) && (
+                <a
+                    aria-label={generateAriaLabel(SocialNetwork.THREADS)}
+                    className={styles.socialLink}
+                    href={createUrlWithQuery('https://www.threads.net/intent/post', {
+                        text: `${title} ${url}`,
+                    })}
+                    onClick={() => trackSharingEvent(SocialNetwork.THREADS)}
```

Comment:
Should we add `rel=“noopener noreferrer”`?

```suggestion
                    onClick={() => trackSharingEvent(SocialNetwork.THREADS)}
                    rel="noopener noreferrer"
```

Reply:
Good point, added with d119793e361a4e8facd9d9b7c501368b53656a8e.